cypress/e2e/integration-manager/get-openid-token.spec.ts

```diff
@@ -59,7 +59,7 @@ const INTEGRATION_MANAGER_HTML = `
 `;
 
 function openIntegrationManager() {
-    cy.findByRole("tab", { name: "Room info" }).click();
+    cy.findByRole("button", { name: "Room info" }).click();
```

Comment:
It would be appreciated if you would let me know that this (and another one below) change meant `<BaseCard>` was not regarded as `tabpanel`. Thanks in advance.